cdap-ui/app/cdap/components/Administration/index.js

```diff
@@ -29,6 +29,7 @@ require('./Administration.scss');
 
 const PREFIX = 'features.Administration';
 const WAITTIME_FOR_ALTERNATE_STATUS = 10000;
+export const SYSTEM_NAMESPACE = 'system';
```

Comment:
would it be better to export this from `global-constants`?

Reply:
The reasoning was to have "admin" level stuff in administration. I could move it to global-constants and change all the references. Don't have a strong opinion about that.

Reply:
It might be better for it to be in global-contants for visibility, since this constant is being used outside Administration components as well